MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -88,7 +88,7 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
                       userAccountResult: MSALNativeAuthUserAccountResult) async -> RefreshTokenCredentialControllerResponse {
```

Comment:
Is it not possible to completely use existing logic? I.e. why do we as native auth need our own refresh token logic?

Even if we do need our own function for this, why do we need specific native auth classes for tokens and account result? Native auth tokens are no different than browser-delegated tokens, or accounts. so can both flows not use the same classes?